src/runtime_src/core/pcie/linux/shim.cpp

```diff
@@ -2149,8 +2149,10 @@ xclDeviceHandle
 xclOpen(unsigned int deviceIndex, const char*, xclVerbosityLevel)
 {
     if(pcidev::get_dev_total() <= deviceIndex) {
+      if(XRT_INFO <= xrt_core::config::get_verbosity()) {
         printf("Cannot find index %u \n", deviceIndex);
-        return nullptr;
+      }
+      return nullptr;
```

Comment:
Can you  use xrt_logmsg?

Reply:
xrt_logmsg is not available in the stub function xclOpen. 
xrt_logmsg is a private member method of xrt_core::xocl::shim class. As this a negative scenario, we may not want to create an object of xocl::shim here.

Reply:
Ok. can you use xrt_core::message::send then?

Reply:
Yes, added.